src/js/eva-config.js

```diff
@@ -58,13 +58,15 @@ function getSpeciesList() {
     //Use assembly accession to de-duplicate common accessions between the
     //browsable assemblies list and the accessioned assemblies list
     return _.uniq(getEVASpeciesList().concat(getAccessionedSpeciesList()),
-                    function(listItem, key, unused) { return listItem.assemblyAccession; });
+                    function(listItem, key, unused) { return [listItem.taxonomyId, listItem.assemblyAccession].join(); });
 }
 
 function deDuplicatedSpeciesList(speciesList) {
     //In the case of multiple assemblies like Grch37 and Grch37.p13 in the species list, use only the first one since
     //they both have the same assembly name and variants from either assembly reside within the same database
-    speciesList = _.sortBy(speciesList, function(listItem) { return listItem.assemblyAccession; });
+    speciesList = _.sortBy(speciesList, function(listItem) {
+                                            return [listItem.assemblyAccession,
+                                                    listItem.taxonomyEvaName.toLowerCase()].join(); });
```

Comment:
any reason why you use the taxonomyEvaName here and the taxonomyId in the other function?

Reply:
The fix here is just a cosmetic fix for the "Variant Browser" drop down to retain one taxonomy "Rice" over the other "Rice (Japanese)" in the sort order when "uniq" is [applied here](https://github.com/EBIvariation/eva-web/pull/145/files#diff-f97583763e7f9dd10a38ed662694f22aR97). 

However, in the deDuplicatedSpeciesList function, taxonomyId is used to deduplicate across multiple species because there could be completely different species that share the same assembly code (for ex: assembly code 20 because these are mostly nonsensical). While, in theory, taxonomyEvaName could still be used here and have the same effect, taxonomyId was chosen because it is an ID as opposed to a string.